### DISCUSSION TITLE
Welcome to 100_Days_Of_Node Discussions!

### DISCUSSION THREAD

**Discussion by Pha12thom** [Pha12thom/100_Days_Of_Node#1] [Announcements]
I'm learning node.js for 100 days, by writing the solution of each single day in this repo, would you be glad to see the 100 challenges to learn more? Follow up with me.

**Comment by Pha12thom** [comment]
I'm learning node.js for 100 days, by writing the solution of each single day in this repo, would you be glad to see the 100 challenges to learn more? Follow up with me.

**Reply by milugoG** [reply]
Good idea

**Comment by milugoG** [comment]
Maybe it'll add more
